inference.py

```diff
@@ -118,6 +143,27 @@ def segmentation(raster, clip_gpkg, model, sample_size, num_bands, device):
                 sample = totensor_transform(sample)
                 inputs = sample['sat_img'].unsqueeze_(0)
                 inputs = inputs.to(device)
+
+                if inputs.shape[1] == 4:
```

Comment:
Seems to be a duplicate of what's at lines 78-97. Could this be put in a function?

Reply:
yeah maybe in the next release